src/shared-components/dropdown/index.js

```diff
@@ -7,7 +7,10 @@ import allowNullPropType from '../../utils/allowNullPropType';
 
 const defaultProps = {
   textAlign: 'left',
+  /* eslint-disable  @typescript-eslint/no-empty-function */
   onChange() {},
```

Comment:
To avoid this (on the other touched files too) you can do:

```suggestion
  onChange: () => undefined,
```